server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -274,6 +288,18 @@ synchronized void setManagerState(ManagerState newState) {
     }
   }
 
+  private void convertPropsToSingleNode() {
+    try {
+      log.info("Starting property conversion");
+      var context = getContext();
+      ConfigPropertyUpgrader configUpgrader = new ConfigPropertyUpgrader();
+      configUpgrader.doUpgrade(context.getInstanceID(), context.getZooReaderWriter());
+      log.info("Completed property conversion");
+    } catch (Exception ex) {
+      throw new IllegalStateException("Failed to convert properties to single node format", ex);
+    }
+  }
+
```

Comment:
If this code is in fact redundant, maybe it could be deleted.

```suggestion
```

Reply:
Removed in e8cbf539bd